dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04330CreateSystemWorkflow.java

```diff
@@ -0,0 +1,215 @@
+
+
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationFeature;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.ObjectMapper;
+import com.dotcms.util.ConversionUtils;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.startup.StartupTask;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import org.apache.velocity.util.ClassUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * This upgrade task will creates the system workflow
+ *
+ * @author Jose Orsini
+ * @version 4.3.0
+ * @since Jan 11, 2018
+ *
+ */
+public class Task04330CreateSystemWorkflow implements StartupTask {
+
+
+    public    static final String SYSTEMWORKFLOW_JSON_PATH = "com/dotmarketing/startup/runonce/json/systemworkflow.json";
+    protected static final String SELECT_SCHEME_SQL        = "select * from workflow_scheme where id = ?";
+    protected static final String INSERT_SCHEME            = "insert into workflow_scheme (id, name, description, archived, mandatory, entry_action_id, default_scheme, mod_date) values (?,?,?,?,?,?,?,?)";
+    protected static final String INSERT_STEP              = "insert into workflow_step (id, name, scheme_id,my_order,resolved,escalation_enable,escalation_action,escalation_time) values (?, ?, ?, ?, ?, ?, ?, ?) ";
+    protected static final String INSERT_ACTION            = "insert into workflow_action (id, scheme_id, name, condition_to_progress, next_step_id, next_assign, my_order, assignable, commentable, icon, use_role_hierarchy_assign, requires_checkout, show_on) values (?, ?, ?, ?, ?, ?, ?,?, ?, ?,?,?,?)";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 302 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)